### PR TITLE
Added encryption_method and encryption_pbkdf

### PR DESCRIPTION
**Pull request by schubi2**

### CODE REVIEW

File: package/yast2-installation-control.changes [hunk 1]
```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Wed Apr 17 15:19:17 UTC 2024 - Stefan Schubert <schubi@suse.com>
+
+- Added encryption_method and encryption_pbkdf in the product
+  description file (section partitioning/proposal) in order to
+  encrypt the generated partitions.
+- 5.0.1
```
Comment (ancorgs) [inline package/yast2-installation-control.changes:7]: I'm not sure if this changelog entry will pass the army of OBS bots. I suggest to add a bugzilla reference. For example https://bugzilla.suse.com/show_bug.cgi?id=1185291

Comment (schubi2) [reply package/yast2-installation-control.changes:8]: Yes, I have adapted it.

**Comment by schubi2** [comment]
Just for history. This PR is needed for https://github.com/yast/yast-storage-ng/pull/1383

**Review by ancorgs — APPROVED** [review]
LGTM

**Pull request merged**